recipes/numpy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version="1.11.0" %}
+
+package:
+  name: numpy
+  version: {{ version }}
+
+source:
+  fn: numpy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/numpy/numpy-{{ version }}.tar.gz
+  md5: bc56fb9fc2895aa4961802ffbdb31d0b
+  # patches:
+    # - osx-dist.patch          # [osx]
+
+build:
+  number: 0
+  features:
+    - blas_openblas
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - openblas
+  run:
+    - python
+    - openblas
+    - libgfortran   >=3.0.0
```

Comment:
Same story as with SciPy.

> I'm not sure I like this. I would rather we use the `gcc` package for now to build Fortran and OpenMP code and add `libgcc` as a runtime dependency as that is what we have [decided]( https://github.com/pelson/Obvious-CI/pull/61#issuecomment-215355681 ) to do for present. We have not included a Fortran compiler in the docker image because we have not settled this point.

-- https://github.com/conda-forge/staged-recipes/pull/516/files/f840390ba3112f87d60de78e206a5014462087d1#r61809882